### PR TITLE
Add collaborators board

### PR DESCRIPTION
**Pull request by jmbriano**
Hi Barcelona team,

Following the initiative #53, please find in this PR an initial version of the collaborators board. This page describes mainly everything we do (in a very short form) so we can easily share to our community. 

Please comment :) It would be nice to have it merged before next monthly so we can give the green light to announce it during the meeting.

### CODE REVIEW

**Comment by jmbriano** [comment]
Tagging people that showed interest in the initiative in the previous monthlies. @raquelalvear, @loriking, @saljuama

**Comment by daverick** [comment]
Hi @jmbriano & @amaliacardenas,

I know that we need to merge it because we announced it at the last meeting. 
but did my comments have been taken into account?

A+

**Comment by jmbriano** [comment]
@daverick where are those comments you refer to?  I think Amalia did the grammar checks, but I can not find any other comments... 

Can you point us where they are? We can extend the PR if needed

File: processes/initiatives/collaborators-board/collaborations.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+## During the workshops
+### Preparing the space
+This task consists in arriving a bit earlier than the _workshop_ start time to **help the company hosting us** prepare the space. Set up tables and electricity extenders, spread the food and drinks, set up the reception desk, and sometimes receive some early comers. The hosts, specially the new ones, are intrigued by the way everything will work and they want to give the best of them. This task also has a bounding part with the host and a part of giving back some help to them.
+
+### Door person
+This task consists in **receiving the participants** while they arrive. Checking-in the students and coaches using codebar platform, giving a name tag and briefly introducing them to the space (where the food, bathrooms, etc are). For new comers, this person can put her/him in contact to someone helping in the integration (see the next activity for details).
```
[the remark this answers is not in the document]
Comment (amaliacardenas) [reply processes/initiatives/collaborators-board/collaborations.md]: The other thing we can do is have a laptop or a cellphone that has already been logged into the account for the door person to use.

**Pull request merged**